packages/core/utils/src/event-bus/build-event-messages.ts

```diff
@@ -48,27 +13,29 @@ export function composeMessage(
   {
     data,
     service,
-    entity,
+    object,
     action,
     context = {},
     options,
   }: {
-    data: unknown
+    data: any
     service: string
-    entity: string
+    object: string
     action?: string
     context?: Context
-    options?: Record<string, unknown>
+    options?: Record<string, any>
   }
 ): EventBusTypes.Message {
   const act = action || eventName.split(".").pop()
-  if (!action && !Object.values(CommonEvents).includes(act as CommonEvents)) {
+  if (
+    !action /* && !Object.values(CommonEvents).includes(act as CommonEvents)*/
```

Comment:
uncomment?

Reply:
I did comment that just in the sense that this utils could be used by an external user for his own events and I was thinking they might have other actions that they would like to provide here. wdyt?

Reply:
yeah I originally kept it to open a discussion 😂